packages/cli/src/polymer-cli.ts

```diff
@@ -131,7 +131,13 @@ export class PolymerCli {
     this.defaultConfigOptions.extraDependencies =
         this.defaultConfigOptions.extraDependencies || [];
     this.defaultConfigOptions.extraDependencies.unshift(
-        `bower_components${pathSeperator}webcomponentsjs${pathSeperator}*.js`);
+        path.join('!bower_components', 'webcomponentsjs', 'gulpfile.js'));
+    this.defaultConfigOptions.extraDependencies.unshift(
+        path.join('bower_components', 'webcomponentsjs', '*.js'));
+    this.defaultConfigOptions.extraDependencies.unshift(path.join(
+        '!node_modules', '@webcomponents', 'webcomponentsjs', 'gulpfile.js'));
+    this.defaultConfigOptions.extraDependencies.unshift(
+        path.join('node_modules', '@webcomponents', 'webcomponentsjs', '*.js'));
```

Comment:
This doesn't look like it goes into`/bundles`, does it?

Reply:
good point